CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing guide
+
+**Want to contribute? Great!**
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples...
+But first, read this page (including the small print at the end).
+
+## Legal
+
+All original contributions to kas-installer are licensed under the
+[ASL - Apache License](https://www.apache.org/licenses/LICENSE-2.0),
+version 2.0 or later, or, if another license is specified as governing the file or directory being
+modified, such other license.
+
+All contributions are subject to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
+The DCO text is also included verbatim in the [dco.txt](dco.txt) file in the root directory of the repository.
+
+## Reporting an issue
+
+This project uses GitHub issues to manage the issues. Open an issue directly in GitHub.
+
+If you believe you found a bug, and it's likely possible, please indicate a way to reproduce it, what you are seeing and what you would expect to see.
+
+## Before you contribute
+
+To contribute, use GitHub Pull Requests, from your **own** fork.
+
+Also, make sure you have set up your Git authorship correctly:
+
+```
+git config --global user.name "Your Full Name"
+git config --global user.email your.email@example.com
+```
+
+If you use different computers to contribute, please make sure the name is the same on all your computers.
+
+We use this information to acknowledge your contributions in release announcements.
+
+### Code reviews
+
+All submissions, including submissions by project members, need to be reviewed by at least two kas-installer committers before being merged.
+
+[GitHub Pull Request Review Process](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) is followed for every pull request.
+
+### Coding Guidelines
+
+ * We decided to disallow `@author` tags in the Javadoc: they are hard to maintain
+ * Commits should be atomic and semantic. Please properly squash your pull requests before submitting them. Fixup commits can be used temporarily during the review process but things should be squashed at the end to have meaningful commits.
+ We use merge commits so the GitHub Merge button cannot do that for us. If you don't know how to do that, just ask in your pull request, we will be happy to help!
+
+
+### Tests and documentation are not optional
```

Comment:
I don't think this project does that today.

Reply:
Certainly tests are not used here, but typically we ask for newly-support env variables to be added to the template along with a short description.